### DISCUSSION TITLE
Temporary Delay in PR Merging and Issue Assignment Due to Exams

### DISCUSSION THREAD

**Discussion by subhadipbhowmik** [subhadipbhowmik/bio-branch#129] [Announcements]
Hey everyone,

I wanted to let you know that my Mid-Semester Test (MST) exams are currently going on. As a result, there might be a slight delay in merging PRs and labeling or assigning issues. However, I will manage these tasks as soon as possible once I have some time. 

In the meantime, feel free to continue contributing, and I truly appreciate your patience and understanding!

Thank you for your support!  
Shubhadip Bhowmik  
Project Maintainer - BioBranch

**Comment by rajdeepchakraborty-rc** [comment]
@subhadipbhowmik I appreciate the fact that you have reached out regarding this. I will continue working on the issues and wait for your feedback(if required any). All the very best for your exams(though it's easy for you to top 😂).
